units/host_termination.go

```diff
@@ -110,6 +110,9 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 			"message":  "host termination for a non-spawnable distro",
 			"cause":    "programmer error",
```

Comment:
Remove the "programmer error" bit since this is now an allowed workflow, not a bug.